src/less/components/input-group.less

```
@@ -73,15 +62,14 @@
         background-color: @white;
         text-align: center;
         white-space: nowrap;
-        border: solid @input-border-color;
-        border-width: @input-border-width;
-        border-bottom-color: @input-addon-color;
-        border-radius: 2px 2px 0 0;
+        border: 1px solid var(--input-border-color);
+        border-radius: 8px 0 0 8px;
```

use the css variable?
```suggestion
            border-radius: var(--border-radius, 8px) 0 0 var(--border-radius, 8px);
```